proto/iot.proto

```diff
@@ -139,7 +139,7 @@ message ThermostatScheduleBlock {
     //Current system mode of thermostat
     //unit: xbos/iot/HVACMode
     HVACMode mode = 3;
-    //Time of schedule retrieval
+    //Time when schedule block takes effect
     //format: RFC
     string time = 4;
```

Comment:
A couple more comments here:

- RFC stands for "request for comments"; did you mean RFC 3339 ?
- RFC 3339 is a date format; is the idea here that each schedule block would be the complete date that the schedule takes effect? (in which case the "day of week" parameter in the ThermostatSchedule message would be redundant) or would we want to consider an RRULE format (or something else) that conveys periodicity or a time of day?

Reply:
- Oops, yes I should be more specific, updated!
- This field was meant to be equivalent to the "time" field in the existing schedule struct from the bw2 version of the Pelican driver ([link](https://github.com/john-b-yang/bw2-contrib/blob/master/driver/pelican/types/thermSchedule.go#L73)). I believe this time format is meant to reflect the frequency of the event (which for thermostats, would be day of the week + time of day), so the RRule format would be useful.

Reply:
Ok, I think we're on the same page now. Can you change the documentation to reflect the RRule format?

Reply:
Oops, got it. So I'm assuming instead of RFC 3339, I should just state "RRule" for format. I'll revert it!